staging/src/k8s.io/apimachinery/pkg/apis/meta/internalversion/validation/validation.go

```diff
@@ -17,17 +17,19 @@ limitations under the License.
 package validation
 
 import (
+	"fmt"
```

Comment:
nit: we should probably structure the PR commits into:
- feature gate
- API change (together with validation)
- changes to watchcache